getter throws exception:
    exception: OutOfRangeException
    name: location
    callSetter: false
    value: null
    callGetter: true
    expect: null

getter returns set value:
    exception: null
    name: location
    callSetter: true
    value: 'a value'
    callGetter: true
    expect: 'a value'

    